vars/withPipeline.groovy

```diff
@@ -107,65 +106,42 @@ def call(type, String product, String component, Closure body) {
         }
 
         onPR {
-          onTerraformChangeInPR{
-            sectionDeployToEnvironment(
-              appPipelineConfig: pipelineConfig,
-              pipelineCallbacksRunner: callbacksRunner,
-              pipelineType: pipelineType,
-              subscription: subscription.nonProdName,
-              aksSubscription: aksSubscriptions.aat,
-              environment: environment.nonProdName,
-              product: product,
-              component: component,
-              tfPlanOnly: true
-            )
-
+          onTerraformChangeInPR {
             def githubApi = new GithubAPI(this)
-
+            def prCache = githubApi.refreshPRCache() // e.g. demo, perftest, ithc, etc
```

Comment:
so we need the for loop to go through these values because the user could name their branch anything it doesn't have to be an env name. For example: this_is_a_branch would try perform a tf plan against that value thinking its an environment

Reply:
think this may be fine - marking as resolved